apps/stats/models.py

```diff
@@ -88,17 +88,17 @@ class Meta:
 
 class ThemeUpdateCountManager(models.Manager):
 
-    def get_range_days_avg(self, start, end):
-        """Return a a dict of the average number of users (count) over the
-        given range of days, per addons."""
-        averages = (self.values('addon_id')
-                        .filter(date__range=[start, end])
-                        .annotate(avg=models.Avg('count')))
-        # Transform the queryset from a list of dicts
-        #   [{'addon_id': id1, 'count__avg': avg1], {'addon_id': id2, ...
-        # to a dict
-        #   {id1: avg1, id2: avg2, ...}
-        return dict((d['addon_id'], d['avg']) for d in averages)
+    def get_range_days_avg(self, start, end, *extra_fields):
```

Comment:
I could be wrong, but if you simply always return the persona id, then you wouldn't need this `extra_fields` "hack", right? And then you would be able to call `get_range_days_avg` with named parameters also? I don't you you'd need to change anything else though.